File: app/src/main/java/com/quran/labs/androidquran/common/HighlightInfo.java
package com.quran.labs.androidquran.common;

import com.quran.labs.androidquran.ui.helpers.HighlightType;

public class HighlightInfo {
  public final int sura;
  public final int ayah;
  public final HighlightType highlightType;
  public final boolean scrollToAyah;

  public HighlightInfo(int sura, int ayah, HighlightType type, boolean scrollToAyah) {
    this.sura = sura;
    this.ayah = ayah;
    this.highlightType = type;
    this.scrollToAyah = scrollToAyah;
  }
}

File: app/src/main/java/com/quran/labs/androidquran/common/HighlightInfo.kt
package com.quran.labs.androidquran.common

import com.quran.labs.androidquran.ui.helpers.HighlightType

class HighlightInfo(
  val sura: Int,
  val ayah: Int,
  val word: Int,
  val highlightType: HighlightType,
  val scrollToAyah: Boolean
)
